### DISCUSSION TITLE
Full backup of all qso data

### DISCUSSION THREAD

**Discussion by oh1jw** [magicbug/Cloudlog#900] [Ideas / Feature Requests]
There's two ways to take backups, ADIF export and Backup. In the end, the both will give you the same end result: adif-file of active station. 
I think the adif export could stay as it is - (a little extra would be option to choose which station data you would like to export, like there is in adif import) -  but the backup function would be more of a dump of all stored qso data.

**Comment by magicbug** [comment]
It's on the todo list to provide a backup system for all parts of Cloudlog, but to be useful it can't just be a simple exported file it needs some route to restore it as QSOs have links to station profile, qsl cards, eqsl cards etc so needs a fair amount of design and decisions on formats.

This is why at the moment best recommendation I have for people is back up the database and local files reguarly.

**Reply by oh1jw** [reply]
Thanks for info. I hold my horses and in the meantime try to learn more about php.

**Comment by AndreasK79** [comment]
@oh1jw one of your wishes came through. Peter merged my PR. Now you can choose station profile in export as well.

**Reply by oh1jw** [reply]
Thank you. It works fine - this is great! TNX

**Reply by AndreasK79** [reply]
@oh1jw happy to help. It was a quick fix to add, and it is something I had in mind a while back when I added tabs in the export. I never got around to add it until I noticed your wish.
